Configuration.props

```diff
@@ -31,6 +31,7 @@
     <MonoSgenBridgeVersion Condition=" '$(MonoSgenBridgeVersion)' == '' ">5</MonoSgenBridgeVersion>
     <HOME Condition=" '$(HOME)' == '' ">$(HOMEDRIVE)$(HOMEPATH)</HOME>
     <AndroidApiLevel Condition=" '$(AndroidApiLevel)' == '' ">25</AndroidApiLevel>
+    <AndroidPlatformId Condition=" '$(AndroidPlatformId)' == '' ">25</AndroidPlatformId>
```

Comment:
What about making this `$(AndroidApiLevel)` instead so that the existing build documentation (i.e. only having to specify `AndroidApiLevel` and `AndroidFrameworkVersion` to build/override in Configuration.Overrides.props) stays valid?